### PR TITLE
Listcollapse

### PR DESCRIPTION
**Pull request by vladimirk4**

### CODE REVIEW

File: app/routes/board/Controller.js [hunk 1]
```diff
@@ -436,6 +437,19 @@ export default ({ store, ref, get, set }) => {
             listTracker.update(store.get("$list.id"), {
                 edit: true
             });
-        }
+        },
+
+
+        showList(e, { store }) {
```
Comment (mstijak) [inline app/routes/board/Controller.js]: I would use `collapsed` instead of `show`. 
Formatting should be fixed.

**Comment by senpl** [comment]
How this suppose to work? Any progress on this ???

**Comment by vladimirk4** [comment]
@senpl it works on mobile and tablet views. Individual boards are collapsible via a button in a board's upper right corner.

**Comment by senpl** [comment]
I finally got it. **Only** when view is phone or tablet it limits showed lists to single list. Thanks for your contribution.

**Pull request merged**